packages/rum-core/src/domain/view/viewMetrics/trackLargestContentfulPaint.spec.ts

```diff
@@ -78,8 +71,11 @@ describe('trackLargestContentfulPaint', () => {
   })
 
   it('should be discarded if the page is hidden', () => {
+    // stop the current tracking from beforeEach
+    cleanup()
```

Comment:
**💬 suggestion:** ‏same here, remove `startLCPTracking` from `beforeEach` and call it in each test case